fortnite_api/cosmetics/br.py

```diff
@@ -241,14 +243,24 @@ class CosmeticBr(Cosmetic[Dict[str, Any], HTTPClientT]):
         The dynamic pak ID of the cosmetic, if available.
     item_preview_hero_path: Optional[:class:`str`]
         The item preview hero path of the cosmetic, if available.
+
+        .. opt-in:: INCLUDE_PATHS
     display_asset_path: Optional[:class:`str`]
         The display asset path of the cosmetic, if available.
+
+        .. opt-in:: INCLUDE_PATHS
```

Comment:
This attribute seems to not present anymore. I need to check back with Officer about it's status.

Reply:
Officer recommends to keep this for now